FDBLibTLS/verify-test.cpp

```diff
@@ -33,11 +33,14 @@
 #include "FDBLibTLS/FDBLibTLSPolicy.h"
 
 struct FDBLibTLSVerifyTest {
-	FDBLibTLSVerifyTest(std::string input):
-		input(input), valid(false), verify_cert(true), verify_time(true), subject_criteria({}), issuer_criteria({}), root_criteria({}) {};
-	FDBLibTLSVerifyTest(std::string input, bool verify_cert, bool verify_time, std::map<int, Criteria> subject, std::map<int, Criteria> issuer, std::map<int, Criteria> root):
-		input(input), valid(true), verify_cert(verify_cert), verify_time(verify_time), subject_criteria(subject), issuer_criteria(issuer), root_criteria(root) {};
-	~FDBLibTLSVerifyTest() {};
+	FDBLibTLSVerifyTest(std::string input)
+	  : input(input), valid(false), verify_cert(true), verify_time(true), subject_criteria({}), issuer_criteria({}),
```

Comment:
I also prefer having long initializer lists be one item per line.

AllowAllConstructorInitializersOnNextLine=false

Reply:
Setting AllowAllConstructorInitializersOnNextLine to false did not have any effect. I can look for something else.